client/src/component/Button/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button } from '@material-ui/core';
+
+const ButtonComponent = ({ variant, color, children, ...rest }) => (
+  <>
+    <Button variant={variant} color={color} {...rest}>
+      {children}
+    </Button>
+  </>
+);
+const { string, node } = PropTypes;
+
+ButtonComponent.propTypes = {
+  variant: string.isRequired,
```

Comment:
we added default values for variant in the theme.js file, So I think it shouldn't be required